docs/pr-review-guidelines.md

```diff
@@ -0,0 +1,80 @@
+# PR Review Guidelines
+
+Here is a handy list of things to consider when doing a review on any PR.
+
+## Initial Checks
+
+- [ ] **Is it a code PR?** If yes, do they have an issue (no matter how small), linked and attached to the PR, describing the desired change or issue?
+  > As it stands today, we don't have strict rules for non code changing PRs – such as README updates or doc changes
+
+- [ ] **Our PR CI passes**
+
+- [ ] **Have they answered the [PR template](../.github/pull_request_template.md) properly** – provided screenshots, have sane responses to the questions, etc
+
+- [ ] **Does it have tests?** (Refer to [Testing](testing.md) documentation)
+  - [ ] **Have you made sure to test the PR image on your cluster?** (this is critically important if there are permission changes since we only have cluster-admin locally; so if there are permission changes along with it, test with and without those permissions to verify the flow on-cluster)
```

Comment:
I don't know if this is hugely true anymore -- I think given the concept of `npm run dev:start:ext` gives the ability to run the UI with whatever user you want... the backend is the only thing that *needs cluster-admin*.

But probably worth still calling out? Not sure if you want to adjust it or not.